modelci/cli/archive/model_cli.py

```diff
@@ -40,9 +38,8 @@
 @click.option('-q', '--quiet', type=click.BOOL, is_flag=True, help='Only show numeric IDs.')
 def models(name, framework, engine, version, list_all, quiet):
     payload = remove_dict_null({'name': name, 'framework': framework, 'engine': engine, 'version': version})
-    with requests.get(f'http://{SERVER_HOST}:{SERVER_PORT}/api/v1/model/', params=payload) as r:
-        model_list = r.json()
-        model_view([model_list], list_all=list_all, quiet=quiet)
+    model_list = get_models(**payload)
```

Comment:
You should use `requests.get`. Keep in mind, CLI is a another way of frontend, it should not call backend function directly. Otherwise, the CLI can only work in a single PC.